tests/component/files/domain/models/FileMetadataMother.ts

```diff
@@ -143,7 +144,7 @@ export class FileMetadataMother {
       size: FileSizeMother.create(),
       date: {
         type: FakerHelper.fileDateType(),
-        date: FakerHelper.recentDate()
+        date: DateHelper.toISO8601Format(FakerHelper.recentDate())
```

Comment:
In every place of tests you are formatting date to iso 8601 format but the date is just a string now, shouldn't we stick to that in tests also?